### ISSUE TITLE
About the stage2

### ISSUE THREAD

**Issue by BUAADreamer** [RupertLuo/VoCoT#3]
Thanks for you great work!!
We are now reproducing your work, we wonder the test results of the second stage model to verify the pretraining of our model.
Could you give the score of MMBench, SEED and Vstar of stage2 model?

**Comment by Junction4Nako** [comment]
Thanks for your attention to our work! As the second stage of our model is designed for **pre-training** to adapt to the multi-modal interleaved contexts, we do not include any instruction-tuning data during that stage. Therefore, we do not expect the stage2 model to fulfill the quite complex tasks like MMBench and SEED as you mentioned. According to our experience, pre-trained models may not possess the capabilities to perform well on these tasks, so we only evaluate the instruction-tuned (Stage3) model on these tasks.
For the sanity check of the stage2 model, we merely perform manual checking to ask it to generate captions and grounded captions for images.
